app/controllers/anonymous_feedback/explore_controller.rb

```diff
@@ -1,15 +1,18 @@
 require 'support/requests/anonymous/explore'
-require 'gds_api/organisations'
+require 'gds_api/support_api'
 
 class AnonymousFeedback::ExploreController < AuthorisationController
   authorize_resource class: Support::Requests::Anonymous::Explore
 
   def new
     @explore_by_url = Support::Requests::Anonymous::ExploreByUrl.new
-    @explore_by_organisation = Support::Requests::Anonymous::ExploreByOrganisation.new
-    @organisations_list = organisations_api.organisations.map { |org|
-      [org.title, org.details.slug]
-    }
+    @explore_by_organisation = Support::Requests::Anonymous::ExploreByOrganisation.new(organisation: current_user.organisation_slug)
+    @organisations_list = support_api.organisations_list.map do |org|
```

Comment:
this has quite a lot going on - maybe pull out both org list construction, and org title construction into separate methods?